gbpservice/neutron/services/servicechain/plugins/msc/drivers/simplechain_driver.py

```diff
@@ -55,7 +55,8 @@
 cfg.CONF.register_opts(service_chain_opts, "simplechain")
 
 # Service chain API supported Values
-sc_supported_type = [pconst.LOADBALANCER, pconst.FIREWALL]
```

Comment:
We can go ahead and merge the following files into One Convergence NFP's GBP code. However, we need to submit to GBP.
 gbpservice/neutron/services/servicechain/plugins/msc/drivers/simplechain_driver.py 
 gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/heat_node_driver.py